ui/src/copyable_text.tsx

```diff
@@ -80,7 +82,16 @@ export const XCopyableText = ({ model }: { model: CopyableText }) => {
 
   return (
     <div data-test={name} className={multiline ? css.multiContainer : css.compactContainer}>
-      <Fluent.TextField componentRef={ref} value={value} label={label} multiline={multiline} styles={{ root: { width: pc(100) } }} readOnly />
+      <Fluent.TextField 
+        componentRef={ref}
+        value={value}
+        label={label}
+        multiline={multiline}
+        styles={ multiline && height && !height.endsWith('%') 
```

Comment:
Restricting the `height` to only be defined in `px` in docstring should be enough.
```suggestion
        styles={ multiline && height
```